src/cli.ts

```diff
@@ -139,6 +141,24 @@ export function runTranspile(files: string[], options: CliOptions) {
   }
 }
 
+function createCairoProject(filePath: string): void {
+  // create cairo_project.toml
+  const cairoProjectPath = path.join(path.dirname(filePath), 'cairo_project.toml');
+  const warplibRoot = path.join(path.dirname(__dirname), 'warplib');
+  outputFileSync(
+    cairoProjectPath,
+    endent`
+      [crate_roots]
+      root = "."
+      warplib = "${warplibRoot}"
+    `,
+  );
+  // create lib.cairo
+  const libPath = path.join(path.dirname(filePath), '/lib.cairo');
```

Comment:
The slash is automatically inserted by `path.join`
```suggestion
  const libPath = path.join(path.dirname(filePath), 'lib.cairo');
```